drivers/interrupt_controller/intc_plic.c

```diff
@@ -21,15 +21,30 @@
 #include <zephyr/drivers/interrupt_controller/riscv_plic.h>
 #include <zephyr/irq.h>
 
+#define PLIC_BASE_ADDR(n) DT_INST_REG_ADDR(n)
```

Comment:
@ycsin ,

Here you've dropped the _U64 postfix. 
This is breaking some of the 64-bit platforms, since the literal integer coming from device tree will not have "ULL"  and the address value could be truncated